cmd/bnsd/app/app_performance_test.go

```diff
@@ -244,16 +223,11 @@ func (n *Nonce) Query() (int64, error) {
 // you want to rapidly generate many tranasactions without
 // querying the blockchain each time
 func (n *Nonce) Next() (int64, error) {
```

Comment:
Maybe instead of returning `(int64, error)` it can return only `int64` and take `testing.TB` to `Failf` if cannot get the nonce? 
This will allow each client to write happy path without checking the error (and yet fail on one).

Reply:
For a test nonce, yes